Project.toml

```
@@ -28,17 +28,17 @@ cohomCalg_jll = "5558cf25-a90e-53b0-b813-cadaa3ae7ade"
 lib4ti2_jll = "1493ae25-0f90-5c0e-a06c-8c5077d6d66f"
 
 [compat]
-AbstractAlgebra = "0.46.2"
+AbstractAlgebra = "0.47.2"
 AlgebraicSolving = "0.9.0"
 Compat = "4.13.0"
 Distributed = "1.6"
 GAP = "0.15.0"
-Hecke = "0.37.2"
+Hecke = "0.38.5"
 JSON = "^0.20, ^0.21"
 JSON3 = "1.13.2"
 LazyArtifacts = "1.6"
 Markdown = "1.6"
-Nemo = "0.51.1"
+Nemo = "0.52.0"
```

```suggestion
Nemo = "0.52.1"
```
for https://github.com/Nemocas/Nemo.jl/pull/2142